src/js/tech/ChromecastTech.js

```diff
@@ -230,52 +233,92 @@ ChromecastTech = {
+            mediaInfoItem.streamType = this.videojsPlayer.liveTracker && this.videojsPlayer.liveTracker.isLive()
+               ? chrome.cast.media.StreamType.LIVE
+               : chrome.cast.media.StreamType.BUFFERED;
+            mediaInfoItem.tracks = [];
+            mediaInfoItem.activeTrackIds = [];
+
+            return new chrome.cast.media.QueueItem(mediaInfoItem);
+         });
+
+         request = new chrome.cast.media.QueueLoadRequest(queueMediaInfo);
+         request.autoplay = true;
+         request.startIndex = loadSource.startIndex;
+         request.currentTime = loadSource.startTime;
+
+         if (loadSource.credentials) {
+            request.credentials = loadSource.credentials;
+            request.credentialsType = loadSource.credentialsType;
          }
-      }
 
-      if (customData) {
-         mediaInfo.customData = customData;
-      }
+         this._isMediaLoading = true;
+         this._hasPlayedCurrentItem = false;
+         castSessionObj = castSession.getSessionObj();
 
-      this._ui.updateTitle(title);
-      this._ui.updateSubtitle(subtitle);
+         castSessionObj.queueLoad(request, this.onLoadSessionSuccess.bind(this), this._triggerErrorEvent.bind(this));
+      } else {
+         this._queue = null;
+         mediaInfo.entity = loadSource.entity;
+         mediaInfo.contentUrl = loadSource.src;
+         mediaInfo.contentType = loadSource.type;
+
+         mediaInfo.metadata = new chrome.cast.media.GenericMediaMetadata();
+         mediaInfo.metadata.metadataType = chrome.cast.media.MetadataType.GENERIC;
+         mediaInfo.metadata.title = title;
+         mediaInfo.metadata.subtitle = subtitle;
+         mediaInfo.streamType = this.videojsPlayer.liveTracker && this.videojsPlayer.liveTracker.isLive()
+            ? chrome.cast.media.StreamType.LIVE
+            : chrome.cast.media.StreamType.BUFFERED;
+         mediaInfo.tracks = [];
+         mediaInfo.activeTrackIds = [];
+
+         for (i = 0; i < textTrackJsonTracks.length; i++) {
+            mediaInfo.tracks.push(this.generateTrack(textTrackJsonTracks[i], i));
+            if (textTrackJsonTracks[i].mode === 'showing') {
+               mediaInfo.activeTrackIds.push(i);
+            }
+         }
 
-      request = new chrome.cast.media.LoadRequest(mediaInfo);
-      request.autoplay = true;
-      request.currentTime = startTime === undefined ? loadSource.startTime : startTime;
+         if (customData) {
+            mediaInfo.customData = customData;
+         }
 
-      if (loadSource.credentials) {
-         request.credentials = loadSource.credentials;
-         request.credentialsType = loadSource.credentialsType;
-      }
+         this._ui.updateTitle(title);
+         this._ui.updateSubtitle(subtitle);
+         request = new chrome.cast.media.LoadRequest(mediaInfo);
+         request.autoplay = true;
+         request.currentTime = startTime === undefined ? loadSource.startTime : startTime;
```

Comment:
Or this

Reply:
https://github.com/standardbroadcast/videojs-chromecast/pull/2/commits/529213ce782c332924c811fc26e7c9d6bcd0805a

Reply:
https://github.com/standardbroadcast/videojs-chromecast/pull/2/commits/ced8dc2daf5445e36dba98608f2d71ab338c41aa